test/fasttests/test_discrete_distribs.py

```diff
@@ -9,36 +9,34 @@
 
 class TestDiscreteDistribution(unittest.TestCase):
 
-    def test_size_unsigned_long(self):
-        get_unsigned_long_size_cf = c_lib.get_unsigned_long_size
-        get_unsigned_long_size_cf.argtypes = []
-        get_unsigned_long_size_cf.restype = ctypes.c_uint8
-        if os.name == 'nt':
-            self.assertEqual(get_unsigned_long_size_cf(),4)
-        else:
-            self.assertEqual(get_unsigned_long_size_cf(),8)
-
-    def test_size_unsigned_long_long(self):
-        get_unsigned_long_long_size_cf = c_lib.get_unsigned_long_long_size
-        get_unsigned_long_long_size_cf.argtypes = []
-        get_unsigned_long_long_size_cf.restype = ctypes.c_uint8
-        self.assertEqual(get_unsigned_long_long_size_cf(),8)
+    # def test_size_unsigned_long(self):
```

Comment:
Why is this commented out?

Reply:
Not sure, I uncommented it in the latest commit.